ci/provisioning/post_provision_config_nodes.sh

```diff
@@ -73,6 +73,34 @@ dump_repos() {
         done
 }
 
+retry_cmd() {
```

Comment:
Based upon https://github.com/daos-stack/daos/pull/6530/files#diff-c7345873ebf345a4a974743526bb970cc1bd63a8997a452def65e942de7d77e7, I'm wondering if this should just be `timeout_cmd()` without the need for the `if [ "$tries" -gt 0 ]; then; sleep "${DAOS_STACK_RETRY_DELAY_SECONDS:-60}"; fi` code.  @JohnMalmberg, thoughts?

Reply:
Or just simply this:
```
timeout_cmd() {
    local timeout="$1"
    shift
    retry_cmd time timeout "$timeout" "$@"
```